src/uucore/src/lib/features/process.rs

```diff
@@ -119,17 +64,15 @@ impl ChildExt for Child {
 
     fn wait_or_timeout(&mut self, timeout: Duration) -> io::Result<Option<ExitStatus>> {
         if timeout == Duration::from_micros(0) {
-            return self
-                .wait()
-                .map(|status| Some(ExitStatus::from_std_status(status)));
+            return self.wait().map(|status| Some(status));
```

Comment:
ERROR: `cargo clippy`: redundant closure (file:'src/uucore/src/lib/features/process.rs', line:67)

Reply:
Oops, I fixed that but forgot to push 🤦🏻